src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,33 +1,39 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
+def merge(arrA, arrB):
+    merged_arr = []
+    while len(arrA) > 0 and len(arrB) > 0:
+        if arrA[0] < arrB[0]:
+            merged_arr.append(arrA.pop(0))
+        else:
+            merged_arr.append(arrB.pop(0))
+    merged_arr.extend(arrA)
+    merged_arr.extend(arrB)
     return merged_arr
 
```

Comment:
great solution; kept code at a minimum and readable.
can you try to solve it without changing the passed in arr (no pop)